README.md

```diff
@@ -28,15 +28,16 @@ To download ScubaGear:
 
 To import the module, open a new PowerShell 5.1 terminal and navigate to the repository folder. 
 
-Depending on the [PowerShell execution policy](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-5.1), running `Unblock-File` on the ScubaGear folder may be required. See [here](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/unblock-file?view=powershell-5.1) for more information.
-
 Then run:
 
 ```powershell
 .\Setup.ps1 #Installs the required modules
 Import-Module -Name .\PowerShell\ScubaGear #Imports the tool into your session
 ```
 
+> **Note**
+> Starting with release 0.3.0, ScubaGear is signed by a commonly trusted CA.  Depending on the [PowerShell execution policy](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-5.1) of the system running ScubaGear, different steps may be required before running running ScubaGear.  See [PowerShell Execution Policies](#powershell-execution-policies) for more details.
```

Comment:
Typo in the text  - remove running from "...different steps may be required before running running ScubaGear".

Reply:
Good catch.  Done!